### ISSUE TITLE
Save flow file as numpy and visualize it

### ISSUE THREAD

**Issue by RokiaAbdeen** [weiyithu/PV-RAFT#5]
thank you so much for sharing your awesome code

could you please explain how to save the flow file as .npy to visualize it as in visualize.py file

thank you in advance

**Comment by RokiaAbdeen** [comment]
I have saved the predicted flow with shape [batch=1,3,npoints=8192] as follows:
`flow=flow.cpu().detach().numpy #convert tensor to numpy file`
`np.save(file_path,flow)`
then, in visual.py I have tried to upload the saved file with the same code as in visual.py but the shape of the flow is zero
could you explain how could I solve this problem please ..

thanks in advance
